gitlabbot/gitlabbot/util.go

```diff
@@ -104,11 +103,6 @@ func isValidArgs (args []string) bool {
 	}
 
 	// check that owner is only alphanumeric
-	for _, r := range args[0] {
-		if !unicode.IsLower(r) && !unicode.IsNumber(r) {
-			return false
-		}
-	}
-
-	return true
-}
\ No newline at end of file
+	match, _ := regexp.MatchString("^[a-zA-Z0-9_]*$", args[0])
```

Comment:
can subgroups have `-` as well?

Reply:
yeah, here i'm only checking that the owner is alphanumeric, i can add checks for the rest of the path